### DISCUSSION TITLE
How to integrate POS Invoices

### DISCUSSION THREAD

**Discussion by salman-aboholiqah** [lavaloon-eg/ksa_compliance#73] [Q&A]
I'm using Frappe 15, To invoicing The Sales Invoices Is cleared and integrated, But POS Invoices is not?

**Comment by salman-aboholiqah** [comment]
Sorry After closing POS Entry It posted to Sales Invoices

**Comment by mhaggag** [comment]
Hello, Salman. Currently, POS invoices aren't supported in the sense that you can't print a POS invoice with the ZATCA print format (which includes QR). Once they generate a sales invoice, the sales invoice is submitted to ZATCA properly and can be printed.

Supporting POS invoices in online mode is in the roadmap.
